providers/standard/src/airflow/providers/standard/sensors/time_delta.py

```diff
@@ -127,12 +127,7 @@ def execute(self, context: Context) -> bool | NoReturn:
                 raise AirflowSkipException("Skipping due to soft_fail is set to True.") from e
             raise
 
-        # todo: remove backcompat when min airflow version greater than 2.11
-        timeout: int | float | timedelta
-        if _get_airflow_version() >= Version("2.11.0"):
```

Comment:
I don't think we can.
Min airflow version for providers is 2.10 not 2.11

Reply:
Oh, i thought we bumped it. This has to be fixed better then. 

Any timeline when we would be bumping that?

Reply:
The conditional itself seems wrong, i attempted to fix it

Reply:
> Any timeline when we would be bumping that?

To 2.11? We will bump to 3.0 in 6 months. But we might tweak the policy according to community needs.

Reply:
Yeah probably wont be doing 2.11 specifically right?

Reply:
Okay compat seems to work now